src/collectors/common-contexts/disk-avgsz.h

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#ifndef NETDATA_DISK_AVGSZ_H
+#define NETDATA_DISK_AVGSZ_H
+
+#include "common-contexts.h"
+
+typedef struct {
+    RRDSET *st_avgsz;
+    RRDDIM *rd_avgsz_reads;
+    RRDDIM *rd_avgsz_writes;
+} ND_DISK_AVGSZ;
+
+static inline void common_disk_avgsz(ND_DISK_AVGSZ *d, const char *id, const char *name, uint64_t avg_bytes_read, uint64_t avg_bytes_write, int update_every, instance_labels_cb_t cb, void *data) {
+    if(unlikely(!d->st_avgsz)) {
+        d->st_avgsz = rrdset_create_localhost(
+            "disk_avgsz"
+            , id
+            , name
+            , "io"
+            , "disk.avgsz"
+            , "Average Completed I/O Operation Bandwidth"
+            , "KiB/operation"
+            , _COMMON_PLUGIN_NAME
+            , _COMMON_PLUGIN_MODULE_NAME
+            , NETDATA_CHART_PRIO_DISK_AVGSZ
+            , update_every
+            , RRDSET_TYPE_AREA
+        );
+
+        rrdset_flag_set(d->st_avgsz, RRDSET_FLAG_DETAIL);

```

Comment:
@ktsaou Should I open a PR that drops the RRDSET_FLAG_DETAIL flag? AFAICT, it's not used  anywhere.